Sources/Private/CoreAnimation/Layers/PreCompLayer.swift

```diff
@@ -104,36 +103,3 @@ extension PreCompLayer: CustomLayoutLayer {
     contentsLayer.masksToBounds = true
   }
 }
-
-extension KeyframeInterpolator where ValueType == AnimationFrameTime {
-  /// A `KeyframeInterpolator` for the given `timeRemapping` keyframes
-  static func timeRemapping(
-    keyframes timeRemappingKeyframes: KeyframeGroup<LottieVector1D>,
-    context: LayerContext)
-    throws -> KeyframeInterpolator<AnimationFrameTime>
-  {
-    try context.logCompatibilityIssue("""
-      The Core Animation rendering engine partially supports time remapping keyframes,
-      but this is somewhat experimental and has some known issues. Since it doesn't work
```

Comment:
This previous implementation attempted to invert the time remapping keyframes. This happened to work in some simple cases, but didn't work in general.